staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -85,6 +86,46 @@ var (
 		},
 		[]string{},
 	)
+	listEtcd3Count = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_list_etcd3_count",
```

Comment:
`apiserver_list_etcd3_count` -> `apiserver_list_etcd_count`
1. For consistency in metrics.go
2. We only have etcd3 in https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apiserver/pkg/storage so far. So using ectd is not ambiguous currently.

Reply:
we should use 'storage' instead of 'etcd3', we're actually trying to be resilient with these metrics in the (unlikely) case that we one day swap storage backends.

Reply:
Revised to use "storage", even for the metric that is now "Number of storage range queries used to satisfy a LIST request", in the force-push to 7e956ce0f29 .